src/main/java/fi/helsinki/cs/tmc/cli/backend/SettingsIo.java

```diff
@@ -142,7 +142,6 @@ private static boolean requireConfigDirectory(Path configRoot) {
             try {
                 Files.createDirectories(configRoot);
             } catch (Exception e) {
-                //TODO print error to user
```

Comment:
Why did you remove this TODO? I don't think logger.error prints a message to the user or does it?